modules/auxiliary/gather/ldap_esc_vulnerable_cert_finder.rb

```diff
@@ -618,7 +611,7 @@ def print_vulnerable_cert_info
           info = hash[:notes].select { |note| note.start_with?(prefix) }.map { |note| note.delete_prefix(prefix).strip }.join("\n")
           info = nil if info.blank?
 
-          hash[:ca_servers].each do |ca_fqdn, ca_server|
+          hash[:ca_servers].each_value do |ca_server|
```

Comment:
Rubocop really wanted this swapped over; even with the explicit `_` prefix that usually override's rubocops semantics in this scenario

```
modules/auxiliary/gather/ldap_esc_vulnerable_cert_finder.rb:614:11: C: [Corrected] Style/HashEachMethods: Use each_value instead of each and remove the unused _fqdn block argument.
          hash[:ca_servers].each do |_fqdn, ca_server| ...
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```